packages/libz/libz.spec

```diff
@@ -0,0 +1,60 @@
+Name: %{_cross_os}libz
+Version: 1.2.11
+Release: 1%{?dist}
+Summary: Compression and decompression library
+URL: https://www.zlib.net/
+License: zlib and Boost
```

Comment:
It looks like the boost license only applies to some contrib code, dotzlib, which I suspect we don't use.  Do we need that?